cmd/release-controller/sync_verify.go

```diff
@@ -7,13 +7,15 @@ import (
 	"strings"
 	"time"
 
-	releasecontroller "github.com/openshift/release-controller/pkg/release-controller"
-
 	"github.com/blang/semver"
-	imagev1 "github.com/openshift/api/image/v1"
-	citools "github.com/openshift/ci-tools/pkg/release/official"
+
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/klog"
+
```

Comment:
over the years, we have found zero value in actually separating these by spaces.